src/components/reusable/MyTable.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import moment from "moment";
+import PropTypes from "prop-types";
+
+const MyTable = ({ columns, data }) => {
+  const [tableColumns, setTableColumns] = useState([]);
+  const [tableData, setTableData] = useState([]);
+
+  useEffect(() => {
+    setTableData(data);
+    setTableColumns(columns);
+  }, []);
```

Comment:
This [approach is not recommended](https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies), if the props `data` or `columns` changes nothing updates in your table.

Reply:
DONE - you are right and I changed it.